app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_sole_trader_journey.scala.html

```diff
@@ -84,6 +85,25 @@
             classes = "govuk-input--width-20"
         )
 
+       @govukCheckboxes(Checkboxes(
+         fieldset = Some(Fieldset(
+           legend = Some(Legend(
+             content = Text("Business verification check"),
+             classes = "govuk-fieldset__legend--m",
```

Comment:
This causes the heading to look different than the others on this page. Doesn't really matter but could we change it to match? Will probably just need to give it the same classes as the others have `govuk-label govuk-label govuk-!-font-weight-bold`

Reply:
I'll change the classes and have a look.